package.json

```diff
@@ -35,17 +35,24 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
+    "@babel/runtime": "^7.7.4",
```

Comment:
probably some of these were installed during experimentation, but we don't actually use them. Anything that's not the 'react-dates' library can prbly be removed again?